app/uk.gov.hmrc.helptosavestridefrontend/connectors/KeyStoreConnector.scala

```diff
@@ -72,8 +75,8 @@ class KeyStoreConnectorImpl @Inject() (val http:                          WSHttp
         case NonFatal(e) ⇒
           val _ = timerContext.stop()
           metrics.keystoreWriteErrorCounter.inc()
-          logger.warn(s"unexpected error when writing stride-user-info to keystore, error=${e.getMessage}")
-          pagerDutyAlerting.alert("unexpected error when storing stride-user-info to keystore")
+          logger.warn(s"unexpected error when writing UserSessionInfo to keystore, error=${e.getMessage}")
+          pagerDutyAlerting.alert("unexpected error when storing UserSessionInfo to keystore")
```

Comment:
we need to remember to update `alert-config` with these pager duty alert messages